ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/cache/TracingCacheManager.java

```diff
@@ -0,0 +1,66 @@
+package org.hyperledger.besu.ethereum.api.cache;
+
+import static org.ehcache.config.builders.CacheConfigurationBuilder.newCacheConfigurationBuilder;
+import static org.ehcache.config.builders.CacheManagerBuilder.newCacheManagerBuilder;
+import static org.ehcache.config.builders.ResourcePoolsBuilder.heap;
+
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.processor.BlockTrace;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.processor.BlockTracer;
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.ethereum.debug.TraceOptions;
+import org.hyperledger.besu.ethereum.vm.DebugOperationTracer;
+import org.hyperledger.besu.plugin.BesuContext;
+import org.hyperledger.besu.plugin.data.PropagatedBlockContext;
+import org.hyperledger.besu.plugin.services.BesuEvents;
+
+import java.util.Optional;
+import java.util.function.Supplier;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.ehcache.Cache;
+import org.ehcache.CacheManager;
+
+public class TracingCacheManager {
```

Comment:
Can tests target this class directly as unit test?

Reply:
Sure.